deploy/ubuntu-pgsql.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+sudo apt update;
+sudo apt install -y postgresql postgresql-contrib
+cd /ias
```

Comment:
Why don't you enable/start PG anymore?

Reply:
Could you add a comment about that to avoid possible confusion during further upgrades of those files?